src/SFA.DAS.ApprenticeAccounts.Database/SFA.DAS.ApprenticeAccounts.Database.sqlproj

```diff
@@ -69,6 +69,9 @@
     <Build Include="Schemas\DashboardReporting.sql" />
     <Build Include="Tables\ClientOutboxData.sql" />
     <Build Include="Tables\OutboxData.sql" />
+    <Build Include="Tables\ApprenticePreferences.sql" />
+    <Build Include="Tables\Preference.sql" />
+    <None Include="PostDeployment\AddDefaultNotificationsPreferences.sql" />
```

Comment:
does this post deploy need to be a Build type as well?

Reply:
Let me know of the change I have added to the database post deployment files is ok